README.md

```diff
@@ -113,11 +113,27 @@ DREAM_VERBOSITY
     See option --verbosity.
 ```
 
+Note that in order for the command-line arguments to be recognized by Dream, and not by the build tool, it is necessary to prepend them with a double dash:
+
+```sh
+# esy
+npx esy start -- -p 9000
+# dune
+dune exec ./main.ml -- -p 9000
```

Comment:
I don't think `esy start` exists, and `dune exec` should be called with the name of the executable, or the path with the `.exe` extension.
```suggestion
# esy
npx esy run <my_binary> -- -p 9000
# dune
dune exec <my_binary> -- -p 9000
```

Reply:
Ah you're right, I guess `start` is a project-specific command